### ISSUE TITLE
Create better examples

### ISSUE THREAD

**Issue by letmaik** [covjson/playground#1]
Example coverages should be more meaningful/realistic, but still simple/small.

**Comment by letmaik** [comment]
@jonblower Any idea? I noticed this as a problem repeatedly while showing it to people. The big grid cells look weird for example.

**Comment by jonblower** [comment]
Barbara is working on a sample flooding dataset that could be useful. And we have some coastal-ocean data, which isn't too big but still looks nice. We'd have to check the licence conditions. Any idea on suitable max size?

**Comment by letmaik** [comment]
The maximum size should be such that it comfortably fits into a 
standalone covjson document that can be displayed in the editor. So not 
super big. Up to 300-400 domain objects are probably fine, for a regular 
axis Grid. Other domain types should be way smaller where we can't use 
regular axes.

**Comment by jonblower** [comment]
So I guess that means a 20x20 grid, roughly speaking (sqrt 400)? But for a regular axis grid, you could have as big a grid as you want and it would be the same size in the encoding? Have I misunderstood?

**Comment by letmaik** [comment]
Sure, but the range values have to be in the document as well. I'd 
rather not have URL ranges for the examples, at least not for the majority.

**Comment by jonblower** [comment]
Right. Hard to make it look "real" with such a small dataset but you could pick a pattern, like concentric circles or an "egg box" pattern. Or take something simpler like a gradient fill and perturb it a little.

If you only want a dataset that size then the grid cells are going to appear pretty big on screen. If you want to reduce their geographical extent (to work at local scale rather than continental), you could do something like air quality within a city (but that's likely to be quite noisy), or maybe try to get data on an urban heat island.

**Comment by letmaik** [comment]
I think 20x20 is enough for a temperature grid of one country. If we 
pick UK, then you can also show missing values (where there's ocean). Is 
there any free/public weather dataset that we could downsample? Could be 
average air temperature of a specific day.
